init.lua

```diff
@@ -0,0 +1,576 @@
+--- a starting point for your own personal configuration. You're supposed to fork this repository
+--- and make it your own! There are lots of comments included, some nicer default options, commonly
+--- agreed on keymaps and some plugins. If you don't like anything in here, feel free to delete it!
+--- It's now your config.
+---
+--- The first time you open neovim after installing this file, your plugin manager will be
+--- installed. After it's done it will tell you to run `:BallsInstall` to install your plugins.
+--- After your plugins are installed you need to restart neovim for them to load.
+
+---{{{ Options
+---
+--- If you want to find out more about any of these, simply run `:help '<option>'`.
+--- For example, if you don't know what the `autoindent` option does, run `:help 'autoindent'`.
+
+-- Indentation
+vim.opt.autoindent = true
+vim.opt.breakindent = true
+vim.opt.breakindentopt = { shift = 2, list = 2 }
+vim.opt.smartindent = true
+vim.opt.shiftwidth = 4
+vim.opt.tabstop = vim.o.shiftwidth -- Make sure 'tabstop' and 'shiftwidth' have the same value
+vim.opt.wrap = true
+
+-- Command completion
+vim.opt.wildoptions = { "fuzzy", "pum" }
+vim.opt.path = { ".", "**" }
+
+-- UI
+if vim.o.textwidth == 0 then
+  vim.opt.colorcolumn = "101"
+else
+  vim.opt.colorcolumn = tostring(vim.o.textwidth + 1)
+end
+
+vim.opt.cursorline = true
+vim.opt.laststatus = 3
+vim.opt.number = true
+vim.opt.relativenumber = true
+vim.opt.scrolloff = 10
+vim.opt.signcolumn = "yes"
+vim.opt.splitbelow = true
+vim.opt.splitright = true
+vim.opt.termguicolors = true
+
+-- Search
+vim.opt.hlsearch = false
+vim.opt.incsearch = true
+vim.opt.ignorecase = true
+vim.opt.smartcase = true
+vim.opt.wildignorecase = true
+
+-- Ripgrep (if available)
+if vim.fn.executable("rg") == 1 then
+  vim.opt.grepformat = "%f:%l:%c:%m"
+  vim.opt.grepprg = "rg --vimgrep -i"
+end
+
+-- Undo history
+vim.opt.undodir = vim.fs.joinpath(vim.fn.stdpath("state"), "undodir")
+vim.opt.undofile = true
+
+-- Fallback colorscheme
+vim.cmd.colorscheme("habamax")
+
+-- Diagnotics
+vim.diagnostic.config({
+  underline = false,
+  severity_sort = true,
+})
+
+---}}}
+
+---{{{ Keymaps
+
+--- This is your leader key.
+---
+--- It will be used for most of your custom keymaps to ensure they don't conflict with anything
+--- that's builtin. It is set to the space key here, but feel free to change it! Many people like
+--- the `,` key as well.
+---
+--- See `:help mapleader` for more information.
+vim.g.mapleader = " "
+
+--- Wrapper function for custom keymaps.
+---
+--- See `:help vim.keymap.set()` for more information.
+local map = function(modes, keys, action, desc)
+  vim.keymap.set(modes, keys, action, { silent = true, desc = desc })
+end
+
+-- File explorer
+map("n", "<Leader>e", vim.cmd.Ex, "Opens netrw.")
+
+-- These two override (in my opinion) bad defaults.
+map("n", "x", "\"_x", "Deletes a single character without yanking it.")
+map("n", "Y", "y$", "Makes `Y` behave more like `D` and `C`.")
```

Comment:
`:h Y` says this tho
```
							*Y*
["x]Y			yank [count] lines [into register x] (synonym for
			yy, |linewise|).
							*Y-default*
			Mapped to "y$" by default. |default-mappings|
```

Reply:
Oh, nice. In vim this is not the case :upside_down_face: